src/pages/resources/reference/push-notifications/templates/tabs/input_box.md

```diff
@@ -0,0 +1,97 @@
+| Sticky | ⛔️ | `adb_sticky` | string | Boolean value represented as a string "true" or "false".<br />When set to false or unset, the notification is automatically dismissed when the user clicks it in the panel. When set to true, the notification persists even when the user clicks it. |
+| Ticker | ⛔️ | `adb_ticker` | string | Sets the "ticker" text, which is sent to accessibility services. Prior to API level 21 (Lollipop), sets the text that is displayed in the status bar when the notification first arrives. |
+| Color - Small Icon | ⛔️ | `adb_clr_icon` | string | Color for the notification's small icon.<br />Represented as six character hex, e.g. `00FF00` |
+| Color - Background | ⛔️ | `adb_clr_bg` | string | Color for notification's background.<br />Represented as six character hex, e.g. `00FF00` |
+| Input receiver name | ✅ | `adb_input_receiver` | string | Required, name of the intent action to use when broadcasting the user input. |
+| Input text | ⛔️ | `adb_input_txt` | string | Placeholder text for the text input field |
+| Feedback text | ⛔️ | `adb_feedback_txt` | string | Once feedback has been submitted, use this text as the notification's body |
+| Feedback image | ⛔️ | `adb_feedback_img` | string | Once feedback has been submitted, use this as the notification's image |
+
+## Example
+
+<img src="./../assets/input_box.gif" width="50%" height="50%"/>
+
+Below is a sample of what a payload might look like for an input box notification:
+
+```json
+{
+  "message": {
+    "token": "FCM_TOKEN",
+    "android": {
+      "collapse_key": "new message"
+    },
+    "data": {    
+      "adb_version": "1",
+      "adb_title": "Check out the new line of shoes!",
+      "adb_body": "Do you have any favorites?",
+      "adb_sound": "sneakerSqueaker",
+      "adb_image": "https://sneakerland.com/products/shoe1/shoe1.png",
+      "adb_uri": "https://sneakerland.com/newReleases",
+      "adb_icon": "ic_shoe",
+      "adb_a_type": "WEBURL",
+      "adb_template_type": "input",
+      "adb_body_ex": "What brands are you interested in?",
+      "adb_clr_body": "00EE00",
+      "adb_clr_title": "AABBCC",
+      "adb_clr_icon": "123456",
+      "adb_clr_bg": "000000",
+      "adb_input_txt": "Enter your favorite brands",
+      "adb_feedback_txt": "Thanks for helping us provide you with an experience more catered to your tastes!",
+      "adb_feedback_img": "https://sneakerland.com/img/thanks.png",
+      "adb_input_receiver": "developer intent action name"
+    }
+  }
+}
+```
+
+## Usage recommendations
+
+The tables below contain guidelines for your push notification content. **These recommendations help your text and images display reliably across multiple devices.** These values are guidelines only - it is still recommended to test a notification prior to sending it.
```

Comment:
Similar comment for the it is recommended as above: "... guidelines only - you should still test a notification..."